.github/workflows/publish-npm.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish
+
+on: 
+  release:
+    types:
+      - "published"
+
+jobs:
+
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - name: Check out repository code
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 16
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Publish latest tag on release
+        if: "!github.event.release.prerelease"
```

Comment:
We can create a release using a git tag, not sure if we can create a prerelease the same way. We may need to update docs about how to tag so it reflects this change. We don't want to need to create the release manually in the gui or use the gh api because it will complicate things.

Reply:
Switched to publishing on pushed tags instead of on published release so we don't have to publish via github.